dist/bin/common

```diff
@@ -61,6 +61,14 @@ if [ ! -x "$JAVACMD" ] ; then
   exit 1
 fi
 
+# parses 1.7, 1.8, 9, etc out of java version "1.8.0_91"
+JAVA_VERSION=$("$JAVACMD" -Xmx512M -version 2>&1 | grep ' version "' | sed 's/.*version "\([0-9]*\)\(\.[0-9]*\)\{0,1\}\(.*\)*"/\1\2/; 1q')
+
+if [ "$JAVA_VERSION" != "1.8" ] ; then
+  echo "Error: Java 8 is required, actual $JAVA_VERSION"
```

Comment:
I think it'd be nice if the message explained that a different version of Java could be set by using the JAVA_HOME environment variable.